docs/cli/edgedb_self/edgedb_self_install.rst

```diff
@@ -0,0 +1,49 @@
+.. _ref_cli_edgedb_self_install:
+
+
+===================
+edgedb self install
+===================
+
+Install the EdgeDB CLI tools and potentially set up the server.
+
+.. cli:synopsis::
+
+    edgedb self install [OPTIONS]
+
+
+Description
+===========
+
+``edgedb self install`` is a terminal command used to install the CLI
```

Comment:
Maybe mention something along the lines of "if you downloaded the binary manually to a temporary location, this is how you install it into standard location"

Reply:
Updated.